frappe/public/js/frappe/form/controls/text_editor.js

```diff
@@ -208,7 +208,7 @@ frappe.ui.form.ControlTextEditor = class ControlTextEditor extends frappe.ui.for
 			},
 			theme: this.df.theme || "snow",
 			readOnly: this.disabled,
-			bounds: this.quill_container[0],
+			bounds: document.body,
```

Comment:
This causes quill to overflow everywhere. The actual content also :pensive:

Reply:
Argh sorry, I should've tested more! 😄